### PR TITLE
Update sprite, backdrop and extension buttons

### PR DESCRIPTION
**Pull request by paulkaplan**
### Resolves

_What Github issue does this resolve (please include link)?_

First step towards https://github.com/LLK/scratch-gui/issues/1166

Just replace the "add extension", "add sprite" and "add backdrop" buttons with the blue roundy ones. Have not changed the buttons in the asset panel yet.

image

### Proposed Changes

_Describe what this Pull Request does_

Create a new button component that will be used to house the behavior of these new asset buttons. For now it just displays the blue circle button.

### Reason for Changes

_Explain why these changes should be made_

First step towards https://github.com/LLK/scratch-gui/issues/1166, but wanted to get this first one out per @thisandagain comment.

### Test Coverage

_Please show how you have added tests to cover your changes_

Updated the integration tests to click by xpath instead of text, because the text is now gone.

### CODE REVIEW

**Comment by thisandagain** [comment]
Thanks for turning that around so quickly @paulkaplan 👍

File: src/components/gui/gui.jsx [hunk 1]
```diff
@@ -108,14 +98,18 @@ const GUIComponent = props => {
...
                                 </Box>
                                 <Box className={styles.extensionButtonContainer}>
-                                    <IconButton
+                                    <button
                                         className={classNames(styles.extensionButton, {
                                             [styles.hidden]: !enableExtensions
                                         })}
-                                        img={addExtensionIcon}
-                                        title={addExtensionMessage}
+                                        title={'Add Extension'}
```
Comment (chrisgarrity) [inline src/components/gui/gui.jsx]: Wouldn't we still want this localized? Is it used by screen readers, or is there another attribute that should be there? Or is the only reason for this to have a unique tag for testing?

**Comment by paulkaplan** [comment]
@chrisgarrity thanks for noticing, I updated the PR to put back the localization for those messages. There is a lot more to be done on these menus, including tooltips with text ([here is the full spec](https://github.com/LLK/scratch-gui/issues/1166)).

**Comment by chrisgarrity** [comment]
@paulkaplan the blue outline focus highlighting continues to highlight the sprite and backdrop buttons after the asset is added:
On the backdrop in this example
screen shot 2018-01-31 at 4 57 21 pm
Is that expected? 

Otherwise things look fine.

**Comment by paulkaplan** [comment]
@chrisgarrity fixed. Usually just disabling that would be an accessibility issue, but since we will be adding more interactivity, the active state will
eventually pop the menu open, so I'm not worried about accessibility of
this right now.

**Comment by paulkaplan** [comment]
@chrisgarrity @carljbowman I updated this to replace the buttons in the asset panels with the blue ones as well, per offline convo.

image

image

**Pull request merged**